src/pages/Detail/Detail.js

```diff
@@ -1,6 +1,186 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Detail.scss';
 
-const Detail = () => {};
+const Detail = () => {
+  const [data, setData] = useState({
+    name: '',
+    quantity: '',
+    size: '',
+    color: '',
+  });
+  const [count, setCount] = useState(1);
+  const price = 260_000;
+  const totalPrice = price * count;
+  const handleCount = () => {
+    count > 1 ? setCount(count - 1) : setCount(1);
+  };
+  const onclick = () => {
+    count < 99 ? setCount(count + 1) : setCount(99);
+  };
 
+  const click = event => {
+    setData(prev => ({ ...prev, size: event.target.title }));
+    console.log(event);
+  };
+
+  const handleColor = event => {
+    setData(prev => ({ ...prev, color: event.target.title }));
+    console.log(event);
+  };
+  console.log(data);
+
+  return (
+    <>
+      <div className="detail">
+        <div className="product_img_list">
+          <p id="product_img_1" />
+          {IMG_LIST.map(list => {
+            return (
+              <img
+                key={list.id}
+                className="product_img"
+                src={list.src}
+                alt="soleMates"
+              />
+            );
+          })}
```

Comment:
```suggestion
          {IMG_LIST.map(({id, src}) => {
            return (
              <img
                key={id}
                className="product_img"
                src={src}
                alt="soleMates"
              />
            );
          })}
```
map 사용시에도 구조분해할당을 사용할 수 있습니다!